### ISSUE TITLE
Tokens manager

### ISSUE THREAD

**Issue by kof** [webstudio-is/webstudio#3901]
As a user I want to be able to manage tokens centrally to solve a number of problems:

1. find and cleanup all unused tokens
2. search for a specific token (might be better to be part of global search)
3. see where a token is being used at a glance
4. rename tokens

**Comment by MilanBgd** [comment]
Really like this feature!

**Comment by cdsaa79** [comment]
this is critical, damn very good practical useful point.

**Comment by kaitpw** [comment]
Will there not be editing functionality involved in this? It'd be powerful for the css copy/paste/editing stuff (on the roadmap) to be enabled in this token manager. I guess that'd make it more like a "token editor" tho

**Comment by kof** [comment]
@kaitpw no, this is too much, css editing is in style panel but you would get to ay element you want with that token and change it there

**Comment by 360Creators** [comment]
Maybe something where you can have tokens set for a whole workspace. Or as it is right now, over all webstudio projects.

**Comment by davidaed** [comment]
Definitely love the bulk rename and categories into folder aspect of Bricks builder. With the ability to click any referenced variable and find it's location.

<img width="1591" height="836" alt="Image" src="https://github.com/user-attachments/assets/7672704d-ef79-4dd3-9dac-bb5235be1471" />

**Comment by rankwebsites** [comment]
Yeah this will be very useful feature.

**Comment by vianmora** [comment]
With the new inception feature, if a want to reuse some of my old component, it's a really pain yes, so a good token manager, where a can merge two existing token, could be really helpfull :D
